src/movies.js

```diff
@@ -1,30 +1,110 @@
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors.
 // How could you "clean" a bit this array and make it unified (without duplicates)?
-function getAllDirectors(moviesArray) {}
+function getAllDirectors(moviesArray) {
+  let allDirectors = moviesArray.map((movie) => {
+    return movie.director;
+  });
+  return allDirectors;
+}
```

Comment:
For shorter syntax:

1. when use use arrow function and have only one line of code to return (literally) , you can use below syntax:
```
  let allDirectors = moviesArray.map((movie) => movie.director);
```

2. You can return the result directly. If you are not going to use the variable  `allDirectors`, no need to actually declare it
```
return moviesArray.map((movie) => movie.director);
```